### PR TITLE
Feature/mvp add ga60 context processor

### PR DESCRIPTION
**Pull request by WillTaylor22**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,12 @@
 
 ## Pre-release
 - no ticket - update components for new cookie banner
-
-## [2020.01.22-b] (https://github.com/uktrade/navigator/releases/tag/2020.01.22-b)
```
Comment (eadpearce) [inline CHANGELOG.md]: why removed?

File: app/navigator/settings/base.py [hunk 2]
```diff
@@ -303,23 +300,20 @@
     'NEW_HEADER_FOOTER_ON': env.bool(
         'FEATURE_NEW_HEADER_FOOTER_ENABLED', False
     ),
-    'HEADER_SEARCH_ON': env.bool('FEATURE_HEADER_SEARCH_ENABLED', False),
-    'ENFORCE_STAFF_SSO_ON': env.bool('FEATURE_ENFORCE_STAFF_SSO_ENABLED', False),
+    'HEADER_SEARCH_ON': env.bool('FEATURE_HEADER_SEARCH_ENABLED', False)
```
Comment (eadpearce) [inline app/navigator/settings/base.py]: this feature flag is obsolete now

File: CHANGELOG.md (hunk 1 above)
Comment (WillTaylor22) [reply CHANGELOG.md]: Ah good spot. Looks like a mouse-slip.

File: app/navigator/settings/base.py (hunk 2 above)
Comment (WillTaylor22) [reply app/navigator/settings/base.py]: Thank you.

**Pull request merged**